src/tests/dictionaryConcepts/container/CreateConcept.test.jsx

```diff
@@ -17,7 +17,7 @@ describe('Test suite for dictionary concepts components', () => {
   const props = {
     match: {
       params: {
-        conceptType: 'question',
+        conceptType: 'Question',
```

Comment:
If this changed from Question to Questions, in how many places will you have to change it?

Reply:
I have created a constant for it.